scripts/host-display

```diff
@@ -91,25 +73,24 @@ def _get_com_port():
     """
     Determine from the current bootloader config which COM port is in use
     """
-    (_, stdout, _) = doexec([XEN_CMDLINE, "--get-xen", "console"])
-    m = re.match(r"console=.*(com\d).*", stdout.readline().strip())
+    (_, stdout) = runCmd([XEN_CMDLINE, "--get-xen", "console"], with_stdout=True)
+    m = re.match(r"console=.*(com\d).*", stdout)
```

Comment:
Why isn't the strip needed here anymore?

Reply:
I believe the strip is unnecessary as the regex won't capture the trailing whitespaces.

Reply:
Strip also removes leading whitespace, which the regex doesn't take into account. (I don't know if it's relevant)

Reply:
I tested with this change which works well, but to align with the previous behavior, I will add `strip()`

Reply:
Hi @psafont , I've updated as your comment, could you review again and approve this PR?